### ISSUE TITLE
Precompiling Yao fails under Julia 1.0

### ISSUE THREAD

**Issue by LaszloHars** [QuantumBFS/Yao.jl#128]
What did I miss?

julia> using Yao
[ Info: Precompiling Yao [5872b779-8223-5990-8dd0-5abbb0748c8c]
WARNING: could not import Base.ctranspose into LuxurySparse
WARNING: could not import LinearAlgebra.ctranspose! into LuxurySparse
ERROR: LoadError: LoadError: LoadError: UndefVarError: RowVector not defined
Stacktrace:
 [1] top-level scope at none:0
 [2] include at .\boot.jl:317 [inlined]
 [3] include_relative(::Module, ::String) at .\loading.jl:1038
 [4] include at .\sysimg.jl:29 [inlined]
 [5] include(::String) at C:\Users\309726\.julia\packages\Yao\ZJlWg\src\LuxurySparse\LuxurySparse.jl:1
 [6] top-level scope at none:0
 [7] include at .\boot.jl:317 [inlined]
 [8] include_relative(::Module, ::String) at .\loading.jl:1038
 [9] include at .\sysimg.jl:29 [inlined]
 [10] include(::String) at C:\Users\309726\.julia\packages\Yao\ZJlWg\src\Yao.jl:8
 [11] top-level scope at none:0
 [12] include at .\boot.jl:317 [inlined]
 [13] include_relative(::Module, ::String) at .\loading.jl:1038
 [14] include(::Module, ::String) at .\sysimg.jl:29
 [15] top-level scope at none:2
 [16] eval at .\boot.jl:319 [inlined]
 [17] eval(::Expr) at .\client.jl:389
 [18] top-level scope at .\none:3
in expression starting at C:\Users\309726\.julia\packages\Yao\ZJlWg\src\LuxurySparse\linalg.jl:29
in expression starting at C:\Users\309726\.julia\packages\Yao\ZJlWg\src\LuxurySparse\LuxurySparse.jl:25
in expression starting at C:\Users\309726\.julia\packages\Yao\ZJlWg\src\Yao.jl:37
ERROR: Failed to precompile Yao [5872b779-8223-5990-8dd0-5abbb0748c8c] to C:\Users\309726\.julia\compiled\v1.0\Yao\TDiQQ.ji.
Stacktrace:
 [1] error(::String) at .\error.jl:33
 [2] macro expansion at .\logging.jl:313 [inlined]
 [3] compilecache(::Base.PkgId, ::String) at .\loading.jl:1184
 [4] macro expansion at .\logging.jl:311 [inlined]
 [5] _require(::Base.PkgId) at .\loading.jl:941
 [6] require(::Base.PkgId) at .\loading.jl:852
 [7] macro expansion at .\logging.jl:311 [inlined]
 [8] require(::Module, ::Symbol) at .\loading.jl:834

**Comment by Roger-luo** [comment]
what is your package status? I guess this might because you got the wrong version of LuxurySparse.jl. Could you please type `status`  in the package mode? 

sorry for the instability at the moment, we will have v0.2 for Julia 1.0 by the end of this month. And thx for the issue.

**Comment by LaszloHars** [comment]
Do I have to explicitly add LuxurySparse?

(v1.0) pkg> status
    Status `C:\Users\xxxx\.julia\environments\v1.0\Project.toml`
  [8f4d0f93] Conda v1.0.1
  [a93c6f00] DataFrames v0.13.0
  [c91e804a] Gadfly v0.7.0
  [91a5bcdd] Plots v0.19.2
  [438e738f] PyCall v1.17.1
  [ce6b1742] RDatasets v0.4.0
  [5872b779] Yao v0.1.4

**Comment by Roger-luo** [comment]
We do not support Julia 1.0 for current release versions. If would like to try our `master` branch (which just support v1.0 the other day), you could use the following command

```julia
pkg> add Yao#master
```

please notice, the master branch is the develop branch and it may not be stable at the moment.

**Comment by LaszloHars** [comment]
Thank you! Yao v0.2.0 #master works, I could run the example code from the documentation.

**Comment by Roger-luo** [comment]
I just tag a deprecation fix for Julia 1.0 (v0.6 support is dropped) as `v0.2.0` (since METADATA only allow minor update to drop old Julia release support, we will add other features in v0.3.0 by the end of this month), try it with

```julia
pkg> add Yao#v0.2.0
```

or wait for https://github.com/JuliaLang/METADATA.jl/pull/16898 and simply update your old package

```julia
pkg> up Yao
```

Or just use the master branch (which may not be stable)

```
pkg> add Yao#master
```

#125 is fixed now
